js/components/graph/EdgeGroup.js

```diff
@@ -38,8 +38,8 @@ export default class EdgeGroup extends React.Component {
   render() {
     // Missing edges must be rendered last. The sort
     // method custom sorts a copy of edgesJSON so that all missing edges
-    // are last in the list. Then render based on that list.
-    const edges = this.props.edgesJSON
+    // are last in the list. Then render based on tha
```

Comment:
This is a typo, I might have introduced it when fixing the merge conflicts (keep the "that list.").